src/containers/Validators/historyTab.scss

```diff
@@ -92,3 +92,14 @@
     color: $orange-50;
   }
 }
+
+.history-note {
```

Comment:
Are there existing styles that could apply? 

Try using the truncate mixin to remove some of these properties